server/client_cloud.go

```diff
@@ -66,3 +67,50 @@ func (client jiraCloudClient) GetUserGroups(connection *Connection) ([]*jira.Use
 	}
 	return groups, nil
 }
+
+func (client jiraCloudClient) ListProjects(query string, limit int) (jira.ProjectList, error) {
+	type searchResult struct {
+		Values     jira.ProjectList `json:"values"`
+		StartAt    int              `json:"startAt"`
+		MaxResults int              `json:"maxResults"`
+		Total      int              `json:"total"`
+		IsLast     bool             `json:"isLast"`
+	}
+
+	remaining := 50
```

Comment:
@levb - just to confirm, this value seems to be useful only when `limit` is 0.

Reply:
Yes, the default. The `limit` is not currently used anywhere, we always use -1 to get all projects.

Reply:
Also, when we fetch all, we do it 50 at a time.